src/main/java/Duke.java

```diff
@@ -1,10 +1,180 @@
+
 public class Duke {
+    public static final String DIVIDER = "___________________________________________________________";
+    public static final String INDENTATION = "      ";
+    public static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    public static final String HELLO_MESSAGE_2 = "Hello! I'm Duke, your friendly neighbourhood task manager";
+    public static final String HELLO_MESSAGE_3 = "What can I do for you? :D";
+    public static final String TASK_COMPLETED_MESSAGE = "You've completed the task! Well done!";
+    public static final String ADDED_TO_LIST = "I've added this to your list :D";
+    public static final String DEADLINE_DESCRIPTION_AND_DATE_SPLITTER = " /by ";
+    public static final String EVENT_DESCRIPTION_AND_DATE_TIME_SPLITTER = " /at ";
+    public static final String GOODBYE_MESSAGE = "Bye, hope to see you again soon! :)";
+    public static final String TYPE_SUITABLE_COMMAND_MESSAGE = "Sorry, you are using me wrongly. Please type in a suitable command :)";
+    public static final boolean IS_FINE = true;
+    public static final int TODO_STARTING_INDEX = 5;
+    public static final int DEADLINE_STARTING_INDEX = 9;
+    public static final int EVENT_STARTING_INDEX = 6;
+    public static Task[] tasks = new Task[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printStartingMessage();
+        Scanner in = new Scanner(System.in);
+        String input = in.nextLine();
+        while (!input.equals("bye")) {
+            processInput(input);
+            input = in.nextLine();
+        }
+        printGoodbyeMessage();
+    }
+
+    public static void printIndentationAndDivider() {
+        System.out.print(INDENTATION);
+        System.out.println(DIVIDER);
+    }
+
+    public static void printWordsWithIndentation(String words) {
+        System.out.print(INDENTATION);
+        System.out.println(words);
+    }
+
+    private static void printStartingMessage() {
+        System.out.println("Hello from\n" + LOGO);
+        printIndentationAndDivider();
+        printWordsWithIndentation(HELLO_MESSAGE_2);
+        printWordsWithIndentation(HELLO_MESSAGE_3);
+        printIndentationAndDivider();
+        System.out.println();
+    }
+
+    private static void printGoodbyeMessage() {
+        printIndentationAndDivider();
+        printWordsWithIndentation(GOODBYE_MESSAGE);
+        printIndentationAndDivider();
+    }
+
+    private static void processInput(String input) {
+        switch(input.split(" ")[0].toLowerCase()) {
```

Comment:
Perhaps you could create a variable with a meaningful name to simplify the expression in the switch statement?